packages/devtools/devtools/src/hooks/useSections.tsx

```diff
@@ -2,40 +2,12 @@
 // Copyright 2020 DXOS.org
 //
 
-import {
-  ChartLine,
-  ComputerTower,
-  CreditCard,
-  Database,
-  GearSix,
-  Devices,
-  FireSimple,
-  Gear,
-  Graph,
-  HardDrive,
-  HardDrives,
-  type Icon,
-  IdentificationBadge,
-  Key,
-  Planet,
-  Polygon,
-  Queue,
-  Receipt,
-  Robot,
-  Users,
-  UsersThree,
-  WifiHigh,
-  MagnifyingGlass,
-  Cloud,
-  Function,
-  LineSegments,
-  Flask,
-} from '@phosphor-icons/react';
+import { type IconName } from '@dxos/react-ui';
 
 export type SectionItem = {
   id: string;
   title: string;
-  Icon: Icon;
+  Icon: IconName;
```

Comment:
This was refactored to `iconName` elsewhere, that does seem more apt than `Icon` which would imply it’s still a component.